lib/pbench/test/unit/server/test_endpoint_configure.py

```diff
@@ -56,18 +56,17 @@ def check_config(self, client, server_config, host, my_headers={}):
                 "graphql": f"{uri}/graphql",
                 "controllers_list": f"{uri}/controllers/list",
                 "controllers_months": f"{uri}/controllers/months",
-                "datasets_namespace": f"{uri}/datasets/namespace/",
-                "datasets_values": f"{uri}/datasets/values/",
-                "datasets_list": f"{uri}/datasets/list",
+                "datasets_delete": f"{uri}/datasets/delete",
                 "datasets_detail": f"{uri}/datasets/detail",
+                "datasets_list": f"{uri}/datasets/list",
                 "datasets_metadata": f"{uri}/datasets/metadata",
+                "datasets_namespace": f"{uri}/datasets/namespace/",
                 "datasets_publish": f"{uri}/datasets/publish",
-                "datasets_delete": f"{uri}/datasets/delete",
+                "datasets_values": f"{uri}/datasets/values/",
```

Comment:
Thanks for the explanation.  It would be good to have a code comment to that effect (so some anal-retentive chap like me doesn't come along and make the URI's "consistent").